src/test/scala-2.12/org/apache/james/gatling/simulation/imap/ImapStoreSimulation.scala

```diff
@@ -3,16 +3,21 @@ package org.apache.james.gatling.simulation.imap
 import com.linagora.gatling.imap.PreDef._
 import io.gatling.core.Predef._
 import io.gatling.core.scenario.Simulation
-import org.apache.james.gatling.control.UserCreator
+import org.apache.james.gatling.control.{UserCreator, UserFeeder}
 import org.apache.james.gatling.imap.scenari.ImapStoreScenario
 import org.apache.james.gatling.simulation.{Configuration, HttpSettings}
 
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, Future}
+
 class ImapStoreSimulation extends Simulation {
   private val users = new UserCreator(Configuration.BaseJamesWebAdministrationUrl).createUsersWithInboxAndOutbox(Configuration.UserCount)
```

Comment:
You don't want to create users, you just want to use the existing one already created in the provisioned data.
You can use the users in SimulationOnMailCorpus like it's done on the JMAP simulations

Reply:
Maybe my comment doesn't hold for this PR. But it will have to be taken into consideration in the PR https://github.com/linagora/james-gatling/pull/59